src/DIPS.Xamarin.UI/Controls/Popup/PopupBehavior.cs

```diff
@@ -99,7 +99,7 @@ public bool IsOpen
         /// </summary>
         public void Hide()
         {
-            HidePopup();
```

Comment:
Noticed that this line caused hide overlay to fire twice because IsOpen=false was also det in the HidePopup method. So I changed it to only fire once